airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/PostgresDestination.java

```diff
@@ -48,12 +49,15 @@ public PostgresDestination() {
 
   @Override
   protected Map<String, String> getDefaultConnectionProperties(final JsonNode config) {
-    if (useSsl(config)) {
-      return SSL_JDBC_PARAMETERS;
-    } else {
-      // No need for any parameters if the connection doesn't use SSL
-      return Collections.emptyMap();
+    final Map<String, String> additionalParameters = new HashMap<>();
```

Comment:
just want to confirm that my understanding is correct, in case SSL_KEY is not provided we are going to enable SSL by default, is it expected behavior?

Reply:
yes, this is normal behavior, which I modified for different types of SSL mode. I will add tests that test this.